web/static/js/components/idea_submission_form.jsx

```diff
@@ -65,12 +65,12 @@ export class IdeaSubmissionForm extends Component {
   }
 
   handleAssigneeChange(event) {
-    this.setState({ assigneeId: Number.parseInt(event.target.value, 10) })
+    this.setState({ assignee_id: Number.parseInt(event.target.value, 10) })
   }
 
   render() {
     const { users, stage } = this.props
-    const { assigneeId, body, ideaEntryStarted, category } = this.state
+    const { assignee_id: assigneeId, body, ideaEntryStarted, category } = this.state
```

Comment:
the more i see it, the more the change to a snake_cased `assignee_id` (client-side, at least) seems to complicate things